### DISCUSSION TITLE
pole_vector (gradient values) to orientation (dip, azimuth, polarity)

### DISCUSSION THREAD

**Discussion by hentsd** [pyvista/pyvista#1455] [Q&A]
Hey!

I need to export pvista's surface orientations as dip, azimuth and polarity.
I managed to export them as gradients (G_x, G_y and G_z), but I either haven't found a way to export them as dip, azim and polaity or a way to convert gradient values to dip, azim and polarity.

hentsd

**Comment by RichardScottOZ** [comment]
Have to see an example, but isn't this just geometry then?
